addons/applovin_max/Example/Scenes/main.gd

```diff
@@ -73,8 +83,19 @@ func _attach_ad_listeners():
 	#banner_listener.on_banner_ad_collapsed = Callable(self, "_on_banner_ad_collapsed")	
 	AppLovinMAX.set_banner_ad_listener(banner_listener)
 	
+	# Set mrec callbacks
+	var mrec_listener = AppLovinMAX.BannerAdEventListener.new()
+	mrec_listener.on_ad_loaded = Callable(self, "_on_banner_ad_loaded")
+	mrec_listener.on_ad_load_failed = Callable(self, "_on_banner_ad_load_failed")
+	mrec_listener.on_ad_clicked = Callable(self, "_on_banner_ad_clicked")
```

Comment:
Might as well rename `_on_banner_ad_...` to `_on_adview_ad_...` to be more generic since it's used for both banners and MRECs now